internal/authentication/file_user_provider.go

```diff
@@ -29,6 +29,7 @@ type FileUserProvider struct {
 // UserDetailsModel is the model of user details in the file database.
 type UserDetailsModel struct {
 	HashedPassword string   `yaml:"password" valid:"required"`
+	Name           string   `yaml:"name" valid:"required"`
```

Comment:
I would use `DisplayName` if we decide to agree on my previous comment.

Reply:
Do you envisage that with the file based backend we would introduce keys like `firstname` `lastname` and displayname`?

Again am happy to adjust here if we think it's necessary, though in the case of the file based backend I think if we decided to introduce `firstname` and `lastname` we could still have `name` alongside it and not introduce a breaking change.

Reply:
I don't think firstname and lastname will ever be introduced, but who knows, but at least we won't have a conflict if that happens. Anyway, I think the argument about readability (both for admins and devs) is the most important here.